django_ibmi/compiler.py

```diff
@@ -112,10 +108,7 @@ def as_sql( self, with_limits=True, with_col_aliases=False, subquery=False ):
         return sql, params
     
     def __map23(self, value, field):
```

Comment:
I suspect this is a python 2 / 3 compatibility method and instead we can use zip_longest directly.